journal/src/main/scala/com/evolutiongaming/kafka/journal/Journals.scala

```diff
@@ -86,8 +90,9 @@ object Journals {
     log: Log[F],
     conversionMetrics: Option[ConversionMetrics[F]]
   ): Journals[F] = {
-    implicit val fromAttempt = FromAttempt.lift[F]
-    implicit val fromJsResult = FromJsResult.lift[F]
+    implicit val fromAttempt: FromAttempt[F]   = FromAttempt.lift[F]
+    implicit val fromJsResult: FromJsResult[F] = FromJsResult.lift[F]
+    implicit val jsonCodec: JsonCodec[Try]     = JsonCodec.summon[F].mapK(ToTry.functionK)
```

Comment:
I see a lot of `summon[F].mapK(ToTry.functionK)` calls. Was it really intended to call it manually everywhere?

Reply:
@rtar what do you propose instead ?

Reply:
Big picture:
1. `JsonCodec[Try]` is required to construct `Codec[Events]` in `Events.codecEvents` method. It is not used there explicitly at all, but just passed into the two methods: `Event.codecEvent` and `ScodecHelper.formatCodec`.
2. It is used directly in `ScodecHelper.formatCodec`.
3. It is used implicitly in `Event.codecEvent` to call `Payload.Json.codecJson`.
4. `Payload.Json.codecJson` just calls `ScodecHelper.formatCodec` again.
5. `ScodecHelper.formatCodec` uses `Attempt.fromTry` on every value received from `JsonCodec`.

What I suggest:
1. `Events.codecEvents` etc. does not have to know anything about `Try`. It only needs `Codec[Event]` and `Codec[PayloadMetadata]`.
2. If one does not want to pass them, but construct in `Events` object instead, the only thing which is required is a function `Format[T] => Codec[T]`, or FunctionK[Format, Codec], no `Try` is involved or required.
3. `JsonCodec` may, indeed, provide implementation of that function using `Try`, but, I guess, it belongs somewhere near `JsonCodec`, i.e. in companion object.